rbac/management/principal/view.py

```diff
@@ -25,6 +26,8 @@
 from ..permissions.admin_access import AdminAccessPermission
 
 USERNAMES_KEY = 'usernames'
+SORTORDER_KEY = 'sort_order'
+VALID_SORTORDER_VALUE = ['asc', 'des']
```

Comment:
I believe the keyword for most database order modifiers should be `desc` in this case? It might make it more intuitive.

Reply:
Ah, it looks like that's what it is in the BOP, eh?

Reply:
Right, there is enum in BOP before my changes: https://github.com/RedHatInsights/backoffice-proxy/blob/master/spec.json#L456-L457

I agree that desc is more intuitive. I think it might be worth changing it.